index.js

```diff
@@ -6,13 +6,36 @@ import { minify_sync } from "terser";
 import { globSync } from "glob";
 import { Table } from "console-table-printer";
 
-const KB = 1024;
+const formatsInBytes = {
+  KB: 1024,
+  MB: 1048576,
+  GB: 1073741824,
+  TB: 1099511627776,
+  PB: 1125899906842580
+}
+
+const byteFormats = ['Bytes', 'KB', 'MB', 'GB', 'TB', 'PB', 'EB', 'ZB', 'YB']
+
+/**
+ * @param {number} bytes
+ * @param {number} decimals
+ * @returns {string}
+ */
+const formatBytes = (bytes, decimals = 2) => {
+  if (bytes === 0) return '0 Bytes'
```

Comment:
Same here. I guess you could do `0 ${byteFormats[0]}`

Reply:
Not sure if that is something needed, it's a fixed value so would be less expensive to just have a string there?

Reply:
I guess. But feels nice to have it all driven from the same source. Though long as we're consistent about using abbreviations i don't mind either way

Reply:
I see you did it anyway. Thanks. Let's get this merged 🚀